src/helpers.js

```diff
@@ -20,7 +19,7 @@ export function isSwagger2(spec) {
     return false;
   }
 
-  return startsWith(swaggerVersion, '2');
+  return swaggerVersion[0] === '2';
```

Comment:
Duplicate of https://github.com/swagger-api/swagger-js/pull/2242/files#r709543609